sntools/event.py

```diff
@@ -1,3 +1,6 @@
+import numpy as np
+import awkward as ak
+
```

Comment:
I don’t think you’re actually using these packages in this file?

Reply:
I answered above